### PR TITLE
Add administrative triage role, update moderator role

### PR DESCRIPTION
**Pull request by devinivy**
> ⚠️ please hold off on merge, even once approved

This updates the moderator role and introduces the triage role per the requirements outlined in #1207.  Also updates the basic auth username for the moderator role, which would invalidate existing tokens out there for it (seems appropriate, given that it now has more privileges).  Finally, this introduces the moderator and triage roles to the bsky appview.

### CODE REVIEW

File: packages/bsky/src/api/com/atproto/admin/getRecord.ts [hunk 1]
```diff
@@ -1,11 +1,10 @@
 import { InvalidRequestError } from '@atproto/xrpc-server'
 import { Server } from '../../../../lexicon'
 import AppContext from '../../../../context'
-import { adminVerifier } from '../../../auth'
 
 export default function (server: Server, ctx: AppContext) {
   server.com.atproto.admin.getRecord({
-    auth: adminVerifier(ctx.cfg.adminPassword),
+    auth: ctx.adminVerifier,
```
Comment (dholms) [inline packages/bsky/src/api/com/atproto/admin/getRecord.ts:7]: we call this `triageVerifier` in the pds & `adminVerifier` here. maybe we sync up the terminology?

It looks like all `triageVerifier` does is also ensure that `credentials.triage` is true (which it always is if `credentials.valid` is true) so probably not a change in logic

I think I like the appview semantics better here 🤔 

altho i wish we had a word to differentiate between "admin as in basic auth triage/moderation/admin" & "admin as in basic auth with admin priveleges"

**Review by dholms — APPROVED** [review]
all looks great besides syncing up our terminology on triage v admin

File: packages/bsky/src/api/com/atproto/admin/getRecord.ts (hunk 1 above)
Comment (devinivy) [reply packages/bsky/src/api/com/atproto/admin/getRecord.ts:7]: Yup yup 👍 I should be able to work something sensible out there.

**Comment by devinivy** [comment]
@dholms I ended-up reframing this all as "role-based" auth in both pds and appview, and then moving the role checks into the handlers where necessary.  Let me know what you think.

**Comment by devinivy** [comment]
Cool!  @bnewbold @dholms this is good to go, but holding off on the merge since it does change the moderator token credentials and will require a little coordination to roll-out.

**Pull request merged**